src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import classes from './Pagination.module.css';
+import { Link } from 'react-router-dom'
+
+

```

Comment:
Let's add a comment here describing what this component does, and some of the logic